File: 0009-palindrome-number/0009-palindrome-number.cs
public class Solution {
    public bool IsPalindrome(int x) {
        string s = Convert.ToString(x) ;
        int start = 0 , end = s.Length - 1 ;
        while(start < end){
            if(s[start] != s[end]){
                return false;
            }
            start++;
            end--;
        }
        return true;
    }
}

File: 0009-palindrome-number/0009-palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
        String s = String.valueOf(x);
        String s1 = "" ;
        for(int i = s.length() - 1 ; i >= 0 ; i--){
            s1 += s.charAt(i);
        }
        return s.equals(s1);
    }
}